.erblint-rubocop20240802-77069-ye9hf4

```diff
@@ -0,0 +1,7 @@
+---
+require: standard
```

Comment:
Should this be in this commit?

Reply:
Nope! I was thinking today how can we get these to go into `tmp`...